projects/storefrontlib/cms-components/product/product-intro/product-intro.component.ts

```diff
@@ -13,52 +14,84 @@ import { CurrentProductService } from '../current-product.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ProductIntroComponent implements AfterContentChecked {
-  reviewsTabAvailable = new BehaviorSubject<boolean>(false);
+  product$: Observable<Product | null> =
```

Comment:
Hey @pawelfras I wonder if this should be added to 5.0md? Here from documentation it says functions but I don't see why it would not also apply to public properties
<img width="440" alt="Screen Shot 2022-03-18 at 9 56 55 AM" src="https://user-images.githubusercontent.com/49131189/159016976-5a1b2640-b1c8-4430-a1bb-e071127f9287.png">